modules/claims_api/app/swagger/claims_api/description/v2.md

```diff
@@ -6,14 +6,13 @@ The Benefits Claims API Version 2 lets internal consumers:
 -   Automatically establish an Intent To File (21-0966) in VBMS
 -   Automatically establish a disability compensation claim (21-526EZ) in VBMS
 -   Digitally submit supporting documentation for disability compensation claims
--   Retrieve the active Power of Attorney for a Veteran
+-   Retrieve the active Power of Attorney organization of individual with power of attorney for a Veteran
+-   Automatically establish a power of attorney appointment in VBMS for an accredited organization (VA Form 21-22).
+-   Automatically establish a power of attorney appointment in VBMS for an accredited individual (VA Form 21-22a).
 
 Additional functionality will be added over time.
```

Comment:
This line is crossed-out in the ticket...looks like it should be removed?

Reply:
Darn, yes that is correct, I re compiled the docs and did not push that up yet

Reply:
Ya good catch missed that line, fixing now